packages/frontend/model/window-guardian-object.ts

```diff
@@ -0,0 +1,58 @@
+// This interface is currently a work in progress.
+// Not all attributes will remain and better types will be given as we go along
+interface WindowGuardianObjectConfig {
+    googleAnalytics: any;
+    images: any;
+    libs: any;
+    modules: any;
+    nav: any;
+    ophan: any;
+    page: any;
+    stylesheets: any;
+    switches: any;
+    tests: {
+        renderer: string;
+    };
+}
+
+export interface WindowGuardianObject {
+    app: {
+        data: {
+            page: string;
+            site: string;
+        };
+        cssIDs: string[];
+    };
+    config: WindowGuardianObjectConfig;
+}
+
+const config = {
+    googleAnalytics: null,
+    images: null,
+    libs: null,
+    modules: null,
+    nav: null,
+    ophan: null,
+    page: null,
+    stylesheets: null,
+    switches: null,
+    tests: {
+        renderer: 'new',
+    },
+};
+
+export const makeWindowGuardianObject = (
+    pageSite: {
```

Comment:
Why not just lift page and site into regular parameters?

Reply:
Ok. I was just extending the existing object and didn't want to introduce regressions but I just check and the `window.guardian` is actually not used anywhere for the moment, so it's safe to change the type.

Reply:
Done!